src/apollo/cmake/ImportEigen3.cmake

```diff
@@ -0,0 +1,14 @@
+# - Defines an imported target for Eigen
```

Comment:
Why not merge these contents into `FindEigen3.cmake`? It seems more convenient when using Eigen